### PR TITLE
Update Rubocop config to match style guide.

### PR DESCRIPTION
**Pull request by petesharum**
All changes should be additive. Some config was rearranged to alphabetize the settings.

---
This change is [Reviewable](https://reviewable.io/reviews/barrettclark/rubocop-github/3)

### CODE REVIEW

File: config/default.yml [hunk 1]
```diff
@@ -9,6 +9,87 @@ Bundler/DuplicatedGem:
 Bundler/OrderedGems:
   Enabled: true
```
Comment (barrettclark) [inline config/default.yml:10]: Do you like this? I tend to disable it. I do generally sort gems now, but sometimes I like to keep things in logical groupings.

**Review by barrettclark — COMMENTED** [review]
I don't have any objections to this, just the question about bundler gem order. We can run this through some apps to see how it performs and what sorts of things it complains about that we might not be so concerned about.

Here is the `.rubycop.yml` file that I had crafted for DrawerZero. There are some Rails and some Style settings that I chiseled over time: https://github.com/the-container-store/DrawerZero/blob/master/.rubocop.yml

File: config/default.yml (hunk 1 above)
Comment (petesharum) [reply config/default.yml:10]: tl;dr: IDK

I included this from the parent's config and haven't considered this one individually before now. Optimistically, I could see a predictable ordering making life easier for troubleshooting or a developer that's new to a project. 

I also like to keep things grouped logically with supporting comments, but it's always been a struggle to maintain that over time. When a gem gets stuck into one of the groupings without thought I'm more likely to overlook it later when troubleshooting.

**Comment by petesharum** [comment]
@barrettclark I'll check for differences between your config and this so we can address each item.

**Comment by petesharum** [comment]
@barrettclark I'm curious if we should (at some point) create a Rails style guide. There are arguably more opportunities to create readability issues with Rails than with Ruby alone. 

Outside of Rails, all of the differences between this and the DrawerZero config are addressed by the style guide.

**Pull request merged**